ideascube/widgets.py

```diff
@@ -1,6 +1,101 @@
 from django.forms import widgets
 
 from ideascube.configuration import get_config
+from django.utils.html import mark_safe, escape
+from django.utils.encoding import force_text
+from django.forms.utils import flatatt
+
+import bleach
+
+
+_script_template = '''<script type="text/javascript">tinymce.init({{
+selector: "#{widget_id}",
+inline: true,
+theme: "inlite",
+hidden_input: false,
+menubar: false,
+toolbar: false,
+selection_toolbar: {settings[selection_toolbar]},
+insert_toolbar: {settings[insert_toolbar]},
+plugins: {settings[plugins]}{settings[extra_settings]}
+}});</script>'''
+
+
+class RichTextEntry(widgets.Widget):
+    def __init__(self, *,
+            with_media = False,
+            selection_toolbar=None,
+            attrs=None):
+        super().__init__(attrs=attrs)
+
+        self.authorized_tags = ['p', 'a', 'ul', 'ol', 'li', 'br',
+                                'h2', 'h3', 'h4', 'h5']
+
+        self.authorized_attributes = {
+            'a': ['href', 'title']
+        }
+
+        if not selection_toolbar is False:
+            selection_toolbar = 'false'
+        else:
+            if selection_toolbar in (True, None):
```

Comment:
As I said in a previous commit, maybe we could simply remove the argument and always use `"bold italic | quicklink h2 h3 blockquote"`.
I don't know, waiting for you point of view :)